ktor-client/ktor-client-js-node/js/src/io/ktor/client/engine/js/Node.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.engine.js
+
+import io.ktor.client.engine.js.compatibility.*
+import io.ktor.utils.io.*
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.*
+import org.khronos.webgl.*
+import org.w3c.dom.*
+import org.w3c.fetch.*
+import kotlin.js.Promise
+
+/**
+ * Node JS API wrapper
+ */
+public object Node : JsPlatformApi {
+
+    override fun fetch(url: String): Promise<Response> = nodeFetch(url)
```

Comment:
Do you need this function? Currently, only the second one with the `init: RequestInit` is called